mkdocs/getting_started.md

```diff
@@ -376,11 +369,51 @@ In order to complete this tutorial you'll one of these devices:
     stm32loader -e -w -v -p /dev/tty.usbserial-10 demos/build/stm32f103c8/Debug/blinker.elf.bin
     ```
 
-    Replace `/dev/tty.usbserial-10` with the correct port.
+    Replace `/dev/tty.usbserial-10` with the correct port 
+    name of the device plugged into your computer via USB.
 
     Use `demos/build/stm32f103c8/Debug/blinker.elf.bin` or replace it with any other
     application to be uploaded.
 
+
+!!! question
+
+    Don't know which serial port to use?
+    ### On Linux
+    With the device unplugged, run the below command
+    ```
+    $ ls /dev/ttyUSB*
+    ls: cannot access '/dev/ttyUSB*': No such file or directory
+    ```
+    Plug the device into the USB port, then rerun the command, the device should appear in the result:
+    ```
+    $ ls /dev/ttyUSB*
+    /dev/ttyUSB0
+    ```
+    The device may also be under the name `/dev/ttyACM*`, like below
+    ```
+    $ ls /dev/ttyACM*
+    /dev/ttyACM0
+    ```
+    ### On Mac
+    With the device unplugged, run the below command
+    ```
+    $ ls /dev/tty.usbserial-*
+    zsh: no matches found: /dev/tty.usbserial-*
+    ```
+    Plug the device into the USB port, then rerun the command, the device should appear in the result:
+    ```
+    $ ls /dev/tty.usbserial-*
+    /dev/tty.usbserial-14240
+    ```
+    ### On Windows
+    Open Device Manager, by pressing the Windows key and typing "Device Manager", then pressing enter.
+
+    Once the Device Manager window is open, plug the device in to your computer via USB and expand
+    the `Ports (COM & LPT)` menu. The device should be visible in the list with a COM port like below:
+
```

Comment:
Can you also add a sentence stating that you'd replace the port in the command with "COM3". Its not super clear what exactly you use from the image.

Reply:
Same goes for the others as well.

Reply:
done